recipe/conda_build_config.yaml

```diff
@@ -2,5 +2,9 @@ c_compiler_version:        # [linux]
   - 12                     # [linux]
 cxx_compiler_version:      # [linux]
   - 12                     # [linux]
-arm_variant_type:  # [aarch64]
-  - sbsa           # [aarch64]
+arm_variant_type:          # [aarch64]
+  - sbsa                   # [aarch64]
+cross_target_platform:
+  - linux-64
+  - linux-ppc64le
+  - linux-aarch64
```

Comment:
```suggestion
  - linux-aarch64
sysroot_linux-64:
  - 2.17
sysroot_linux-ppc64le:
  - 2.17
sysroot_linux-aarch64:
  - 2.17
```

Reply:
What's the purpose of this? It's not used

Reply:
Please see comment below